libr/core/linux_heap_glibc.c

```diff
@@ -83,8 +83,12 @@ static void GH(print_main_arena)(RCore *core, GHT m_arena, GH(RHeap_MallocState)
 	PRINTF_BA ("0x%08x\n", (ut32)main_arena->flags);
 	PRINT_GA ("  fastbinsY = {\n");
 
-	for (i = 0, j = 1, k = SZ * 4; i < NFASTBINS - 2; i++, j++, k += SZ * 2) {
-		PRINTF_YA (" Fastbin %02d: ", j);
+	for (i = 0, j = 1, k = SZ * 4; i < NFASTBINS; i++, j++, k += SZ * 2) {
+		if (FASTBIN_IDX_TO_SIZE(j) <= global_max_fast) {
+			PRINTF_YA (" Fastbin %02d\n", j);
+		} else {
+			PRINTF_RA (" Fastbin %02d\n", j);
```

Comment:
We probably should choose a different color here, because red is being used for error/corrupted stuff.

This is how it looks right now:

![screen shot 2017-05-17 at 09 55 21](https://cloud.githubusercontent.com/assets/3370782/26145852/506d0526-3aee-11e7-859a-c63be82b7b44.png)